### PR TITLE
adding qmk firmware for fuji65

### PR DESCRIPTION
**Pull request by CMMS-Freather**
## Description

adding qmk firmware for fuji65 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cmm_studio/fuji65/config.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/* Copyright 2021 REPLACE_WITH_YOUR_NAME
```
Comment (peepeetee) [inline keyboards/cmm_studio/fuji65/config.h:1]: Replace with, well, your name

File: keyboards/cmm_studio/fuji65/fuji65.h [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/* Copyright 2021 REPLACE_WITH_YOUR_NAME
```
Comment (peepeetee) [inline keyboards/cmm_studio/fuji65/fuji65.h:1]: Replace with your name

File: keyboards/cmm_studio/fuji65/rules.mk [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (peepeetee) [inline keyboards/cmm_studio/fuji65/rules.mk:17]: remove line 17

**Comment by peepeetee** [comment]
files for sam/sg81m are also included, is it intended, or is it because you are using your master branch?

**Comment by CMMS-Freather** [comment]
> files for sam/sg81m are also included. is it intended, or is it because you are using your master branch?

its because i am using my master branch. my initial intention was only to add the fuji65 firmware support for this pr

**Comment by peepeetee** [comment]
> its because i am using my master branch. my initial intention was only to add the fuji65 firmware support for this pr

I'd recommend creating a new branch, bring it level with upstream/master, and then checking out the fuji65 files. Do you need any help with that?

**Comment by CMMS-Freather** [comment]
> > its because i am using my master branch. my initial intention was only to add the fuji65 firmware support for this pr
> 
> I'd recommend creating a new branch, bring it level with upstream/master, and then checking out the fuji65 files. Do you need any help with that?

i see what you mean there. I guess i will close this pr and create a new pr for easy convininece. thx for the help

**Comment by peepeetee** [comment]
> > > its because i am using my master branch. my initial intention was only to add the fuji65 firmware support for this pr
> > 
> > 
> > I'd recommend creating a new branch, bring it level with upstream/master, and then checking out the fuji65 files. Do you need any help with that?
> 
> i see what you mean there. I guess i will close this pr and create a new pr for easy convininece. thx for the help


Don't open another PR just for this, you can switch which branch you are merging from within the PR

**Comment by CMMS-Freather** [comment]
> > > > its because i am using my master branch. my initial intention was only to add the fuji65 firmware support for this pr
> > > 
> > > 
> > > I'd recommend creating a new branch, bring it level with upstream/master, and then checking out the fuji65 files. Do you need any help with that?
> > 
> > 
> > i see what you mean there. I guess i will close this pr and create a new pr for easy convininece. thx for the help
> 
> Don't open another PR just for this, you can switch which branch you are merging from within the PR

not too sure how to do it but i forked a branch and reopened the PR now. Is there something I did wrong?

**Comment by peepeetee** [comment]
> > > > > its because i am using my master branch. my initial intention was only to add the fuji65 firmware support for this pr
> > > > 
> > > > 
> > > > I'd recommend creating a new branch, bring it level with upstream/master, and then checking out the fuji65 files. Do you need any help with that?
> > > 
> > > 
> > > i see what you mean there. I guess i will close this pr and create a new pr for easy convininece. thx for the help
> > 
> > 
> > Don't open another PR just for this, you can switch which branch you are merging from within the PR
> 
> not too sure how to do it but i forked a branch and reopened the PR now. Is there something I did wrong?

Sorry, it seems like that I was wrong in my assertion that you could reuse this PR. There seems to be no way to change from which branch you are merging from, your best course of action might indeed be to close this PR and reopen one.

**Pull request closed without merging**